lib/run.js

```diff
@@ -7,30 +7,33 @@ require('newrelic'); // eslint-disable-line global-require
 require('./config/sentry').initializeSentry();
 
 const { redisOptions } = require('./config/redis-info')('probot');
-const { findPrivateKey } = require('probot/lib/private-key');
-const { createProbot } = require('probot');
+const { getPrivateKey } = require('@probot/get-private-key');
+const { Server, Probot } = require('probot');
 const app = require('./configure-robot');
 
 const workers = Number(process.env.WEB_CONCURRENCY || 1);
 
-const probot = createProbot({
-  id: process.env.APP_ID,
-  secret: process.env.WEBHOOK_SECRET,
-  cert: findPrivateKey(),
+const server = new Server({
+  Probot: Probot.defaults({
+    appId: process.env.APP_ID,
+    secret: process.env.WEBHOOK_SECRET,
+    privateKey: getPrivateKey(),
+    redisConfig: redisOptions,
+    logLevel: process.env.LOG_LEVEL,
+  }),
   port: process.env.TUNNEL_PORT || process.env.PORT || 3000,
   webhookPath: '/github/events',
   webhookProxy: process.env.WEBHOOK_PROXY_URL,
-  redisConfig: redisOptions,
 });
 
 /**
  * Start the probot worker.
  */
-function start() {
+async function start() {
   // We are always behind a proxy, but we want the source IP
-  probot.server.set('trust proxy', true);
-  probot.load(app);
-  probot.start();
+  server.expressApp.set('trust proxy', true);
+  await server.load(app);
+  server.start();
```

Comment:
`server.start()` is an async method, too:

https://github.com/probot/probot/blob/4b3ae0eee59c7ecf0348299b9c13fe87e516c740/src/server/server.ts#L69

```suggestion
  await server.start();
```

Reply:
done